charmdir.go

```diff
@@ -214,6 +223,16 @@ func (zp *zipPacker) AddRevision(revision int) error {
 	return err
 }
 
+func (zp *zipPacker) AddVersion(version string) error {
```

Comment:
This can be factored out into a single helper function called AddFile(filename, content string)
zp.AddFile("revision", strconv.Itoa(revision))
zp.AddFile("version", versionString)

Reply:
done